src/debugger/appWorker.ts

```diff
@@ -92,8 +95,12 @@ export class SandboxedAppWorker {
 
     private initializeSandboxAndContext(scriptToRunPath: string): void {
         let scriptToRunModule = new Module(scriptToRunPath);
+        scriptToRunModule.paths = Module._nodeModulePaths(path.dirname(scriptToRunPath));
```

Comment:
This seems to be using private methods. I'm guessing you are doing this because we can't achieve the same effect with public ones. Please add some comments explaining what are you doing here, and how did you figure it out, so it's easy in the future to re-implement this if they change or remove _nodeModulePaths and/or .paths.